src/Arcus.EventGrid.Tests.Integration/appsettings.json

```diff
@@ -1,12 +1,24 @@
 {
   "Arcus": {
-    "EventGrid": {
-      "TopicEndpoint": "#{Arcus_EventGrid_TopicEndpoint}#",
-      "EndpointKey": "#{Arcus_EventGrid_EndpointKey}#"
+    "EventGridEvent": {
```

Comment:
Please align tokens to use our existing naming convention

Reply:
Can you also add them to the build please? I don't see our build yaml being updated

Reply:
No, I updated the library in azure devops.

Reply:
Ok but can we at least align the tokenization naming to align with others?

Reply:
Have actually no idea what you're talking about, or what you want to change 😄 .

Reply:
`Arcus_EventGrid_EndpointKey` -> `EventGrid.EndpointKey`

Reply:
- Don't forget to update the variable group as all your tests are broken now
- Don't forget to remove the `Arcus.` prefix similar to our existing approach